1-Draft/RFCxxxx-PSModulePath.md

```diff
@@ -135,22 +135,12 @@ Change would be to use `$env:PSModulePath`, but prefix with `User` modules path
 
 Note that Windows PowerShell here means both `powershell.exe` and `powershell_ise.exe`.
```

Comment:
What about `isep.exe` (PowerShell ISE Preview)? I still see the occasional user working with that.

Reply:
That project never really went anywhere.  I don't know if we need to really cover that case.